config/assisted-service/kustomization.yaml

```diff
@@ -1,10 +1,10 @@
 resources:
 - scality-secret.yaml
```

Comment:
don't we already work with filesystem?

Reply:
@rwsu mentioned there being a reason for us to have that. Would be great if we could remove it.

Reply:
Removing it.